examples/secrets/values-secret.v2.yaml

```diff
@@ -0,0 +1,81 @@
+# NEVER COMMIT THESE VALUES TO GIT (unless your file only uses generated
+# passwords or only points to files)
+
+# Needed to specify the new format (missing version means old version: 1.0 by default)
+version: 2.0
```

Comment:
Suggest we also include something to the effect of:

`backingStore: vault`

Reply:
Ack